agent/cmd/determined-agent/init.go

```diff
@@ -152,6 +152,6 @@ func registerAgentConfig() {
 	registerInt(flags, name("agent-reconnect-backoff"), defaults.AgentReconnectBackoff,
 		"Time between agent reconnect attempts")
 
-	registerString(flags, name("container-runtime"), defaults.ContainerRuntime,
-		"The container runtime to use")
+	registerString(flags, name("docker-container-runtime"), defaults.DockerContainerRuntime,
```

Comment:
What is this change?

Reply:
change is to make config definition for /etc/determined/agent.yaml file. 
My changes will be undone here after your suggestions below.